boards/arc/qemu_arc/qemu_arc_em.yaml

```diff
@@ -11,3 +11,4 @@ testing:
   ignore_tags:
     - net
     - bluetooth
+vendor: qemu
```

Comment:
We should have `snps` here.

Reply:
this conflicts with the DTS and with all other qemu we have, which have qemu as vendor. Qemu is not vendor specific.

Reply:
> Qemu is not vendor specific.

We are not talking about the emulator itself - but we are talking about the exact Zephyr board (which is vendor specific).

> this conflicts with the DTS

I'm not asking to change the DTS, I'm only asking about the board's `.yaml` file. What's the point of adding filtering by board vendor if we can't actually filter by board vendor? :)

Reply:
My point is that we want to be aligned with what the DTS has, and the DTS has qemu, not snps. So, if the DTS had snps, we wont be having this conversation. If you want snps, change the DTS and we are good.

Reply:
ok, I changed it now, but it would be nice if the DTS compatible is changed as well.

Reply:
Let's not have a different approach to qemu boards for different architectures. I don't see a problem with 'qemu' as vendor.

Reply:
after grepping, I actually found others using vendors:

```
vendor boards/*/*/qemu*yaml
boards/arc/qemu_arc/qemu_arc_em.yaml:vendor: snps
boards/arc/qemu_arc/qemu_arc_hs.yaml:vendor: qemu
boards/arc/qemu_arc/qemu_arc_hs5x.yaml:vendor: qemu
boards/arc/qemu_arc/qemu_arc_hs6x.yaml:vendor: qemu
boards/arc/qemu_arc/qemu_arc_hs_xip.yaml:vendor: qemu
boards/arm/qemu_cortex_a9/qemu_cortex_a9.yaml:vendor: xlnx
boards/arm/qemu_cortex_m3/qemu_cortex_m3.yaml:vendor: ti
boards/arm/qemu_cortex_r5/qemu_cortex_r5.yaml:vendor: xlnx
boards/arm64/qemu_cortex_a53/qemu_cortex_a53.yaml:vendor: qemu
boards/arm64/qemu_cortex_a53/qemu_cortex_a53_smp.yaml:vendor: qemu
boards/arm64/qemu_kvm_arm64/qemu_kvm_arm64.yaml:vendor: qemu
boards/mips/qemu_malta/qemu_malta.yaml:vendor: qemu
boards/mips/qemu_malta/qemu_malta_be.yaml:vendor: qemu
boards/nios2/qemu_nios2/qemu_nios2.yaml:vendor: qemu
boards/x86/qemu_x86/qemu_x86.yaml:vendor: qemu
boards/x86/qemu_x86/qemu_x86_64.yaml:vendor: qemu
boards/xtensa/qemu_xtensa/qemu_xtensa.yaml:vendor: cdns
```